scripts/src/blockchain/transactionBuilder.ts

```diff
@@ -35,8 +44,12 @@ export class TransactionBuilder {
 		return this;
 	}
 
+	public set channel(channle: Channel | undefined) {
```

Comment:
any reason setter was added? the channel should be set on creation only (as it requires fetching sequence id from the network)